src/Symfony/Component/Routing/Loader/YamlFileLoader.php

```diff
@@ -27,7 +27,7 @@
  */
 class YamlFileLoader extends FileLoader
 {
-    private static $availableKeys = array(
+    protected static $availableKeys = array(
```

Comment:
This introduces inheritance extension points, which then have to be maintained forever. And inheritance extension points are the worse to maintain for BC. So -1